src/gsi/apply_scaledepwgts.f90

```diff
@@ -0,0 +1,174 @@
+function fwgtofwvlen (rvlft,rvrgt,rcons,rlen,rinput)
+!$$$  subprogram documentation block
+! 
+! subprogram: fwgtofwvlen
+!  
+!  program history:
+!
+!  28-03-2018    T. Lei and D. Kleist   consoliated and added codes 
```

Comment:
@Wangy1111 Would you put this log data to the original location (in the log history for the whole file) rather than just this subroutine? Of course, if you need, you can add more log data into it.  I may try to add the missing log data for the initial version by D. Kleist.

Reply:
@TingLei-NOAA Thank you. 
I have put the log for all subroutines, not only for the function fwgtofwvlen. I am not sure if the original format still obeys the GSI code standard. If it is so, I can revert it.

Reply:
@Wangy1111 That log item is a very reduced /casual log for the whole file and I don't think it violated any standards. If it did, violate any standards,  please let me know, I will modify it appropriately.

Reply:
It looks good to me.

Reply:
@TingLei-NOAA As I didn't find any codes with the log at the header (usually inside the subroutine), I put the original log into each subroutine. Sorry for any inconvenience. Please let me know if any comments. Thank you.

Reply:
Minor comment.  I checked program logs in other GSI source code files.  Most use the format YYYY-MM-DD.   For example, `pcgsoi.f90` has
```
! program history log:                                                                                      
!   2005-05-17  Yanqiu zhu - wrap pcgsoi and its tangent linear pcgsoi_tl into one module                   
!   2005-11-21  Derber - remove interface                                                                   
!   2008-11-26  Todling - remove pcgsoi_tl                  
```           
File `apply_scaledepwgts.f90` has DD-MM-YYYY.   
```
!  program history:                                                                                         
!                                                                                                           
!  28-03-2018    T. Lei and D. Kleist   consoliated and added codes                                         
!                                       for the scale dependent scale localization scheme 
```
Why the change?   

This is a very minor comment.  It has nothing to do with code functionality.   That said, it would be nice to be consistent.

Reply:
@RussTreadon-NOAA Thank you for the comment. I just updated it to YYYY-MM-DD format.